runtime/browser/runtime_geolocation_permission_context.h

```diff
@@ -26,7 +26,7 @@ class RuntimeGeolocationPermissionContext
   virtual void RequestGeolocationPermission(
       content::WebContents* web_contents,
       const GURL& requesting_frame,
-      base::Callback<void(content::PermissionStatus)> result_callback);
+      base::Callback<void(bool)> result_callback);
```

Comment:
This is not what upstream is moving to. I actually changed bool -> PermissionStatus, it was bool before. My quick look is that we just need to modify the ultimate piece going to Java from bool to enum.